pkg/volume/cinder/attacher.go

```diff
@@ -88,6 +88,7 @@ func (attacher *cinderDiskAttacher) Attach(spec *volume.Spec, nodeName types.Nod
 		glog.Warningf(
 			"Error checking if volume (%q) is already attached to current instance (%q). Will continue and try attach anyway. err=%v",
 			volumeID, instanceid, err)
+		return "", err
```

Comment:
@jsafrane Are you familiar with the implications of continuing to attach here?  Should we keep the old behavior or change it to return an error?